### PR TITLE
[ASAA-177] Air Quality Data

### PR DESCRIPTION
**Pull request by MMurrayAtBR**
### [[ASAA-177] Air Quality Data](https://bottlerocketstudios.atlassian.net/browse/ASAA-177)

- Updated AirQualityEnum with when function to transform AQI return to an AirQualityEnum
- Added KMP Launchpad Google Utils library to fetch Air Quality details
- Added HomeViewModel `getAirQualityDetails()` to call AirQualityApiService

### CODE REVIEW

File: shared/src/commonMain/kotlin/com/br/kmpdemo/viewmodels/HomeViewModel.kt [hunk 1]
```diff
@@ -140,6 +141,19 @@ class HomeViewModel : BaseViewModel() {
                 .onSuccess { realTimeResponse.value = it }
                 .onFailure { Logger.e("[getRealTimeForecasts]") { "Failure: ${it.message}" } }
         }
+
+    // TODO: ASAA-196 Add details for AirQualityWidget "See More" navigation
+    private fun getAirQualityDetails(location: UserLocation) =
+        viewModelScope.launch {
```
Comment (BottleRocket-Colin) [inline shared/src/commonMain/kotlin/com/br/kmpdemo/viewmodels/HomeViewModel.kt:147]: do we have launchIO in this project?

File: compose/src/commonMain/kotlin/com/br/kmpdemo/compose/ui/weatherDetails/airQuality/AirQualityEnum.kt [hunk 2]
```diff
@@ -11,4 +11,15 @@ enum class AirQualityEnum(val level: Float, val quality: StringResource) {
     PURPLE(0.5f, SharedRes.strings.aqi_5), // 201-300
     MAROON(0.6f, SharedRes.strings.aqi_6), // Over 300
     UNKNOWN(0.0f, SharedRes.strings.unknown_aqi)
+}
+
+fun Int.getAirQualityEnum(): AirQualityEnum {
```
Comment (BottleRocket-Colin) [inline compose/src/commonMain/kotlin/com/br/kmpdemo/compose/ui/weatherDetails/airQuality/AirQualityEnum.kt:16]: could be expression function

**Review by BottleRocket-Colin — CHANGES_REQUESTED** [review]
Couple of very minor changes.

File: shared/src/commonMain/kotlin/com/br/kmpdemo/viewmodels/HomeViewModel.kt (hunk 1 above)
Comment (MMurrayAtBR) [reply shared/src/commonMain/kotlin/com/br/kmpdemo/viewmodels/HomeViewModel.kt:147]: We didn't, but we do now!

**Review by BottleRocket-Colin — APPROVED** [review]

**Pull request merged**